### DISCUSSION TITLE
Genealogy feature?

### DISCUSSION THREAD

**Discussion by smogger** [immich-app/immich#5144] [General]
Avid user of Immich, and really love how the project evolves by every releases. 

### **Behind story:** 
Got over 400 GB of old scans till the late 1800, early 1900 of my ancestor (1316 folders, 36000+ files), organized by folders/dates taken, but not yet imported to Immich as waiting for Meta data editor, and some other features to be done before I do this big jump. (till then I archived / syncthing the photos, and use Immich as my phone photo organizer for new photos.

### **Base features:**
Usually what are we storing on Google Photos, Immich, etc? My best bet are **_family photos_**.
- We have face detection (or constantly improving)
- We have (will have) good meta data editor 
- Most of our photos are about events/family members, with geotagging feature

### **Idea:**
- Implement a "basic" genealogy (family tree) feature, where we can simply tag those family members, based on face detection.
- Based on Face detection we can also add some additional information about that specific person (birth date, city born which can be also visible on a map) - toggle between "photo taken" or "Person born"

I know this is a bit off, what Immich is trying to solve, but based on how fast this project improves day by day, would be great to see this direction too.

**Comment by bo0tzz** [comment]
This is not in scope for Immich. Maybe an addon for some genealogy software that connects it to Immich would do the trick?
